app/views/registrations/new.html.erb

```diff
@@ -34,6 +34,13 @@
       </div>
     </div>
 
-    <%= form.button "Sign Up", type: "submit", data: { test_id: "sign_up_button" }, class: "w-full bg-red py-4 text-white font-black italic rounded-[10px] uppercase" %>
+    <%= form.button(
+      "Sign Up",
+      data: { test_id: "sign_up_button" },
+      class: [
+        "w-full bg-red py-4 text-white font-black italic rounded-[10px] uppercase",
+        "disabled:bg-gray-400 disabled:cursor-not-allowed"
+      ]
+    ) %>
```

Comment:
Not caused by this PR but fields with errors input gets shortened
<img width="502" alt="image" src="https://github.com/user-attachments/assets/91e9366b-fb03-420a-a63f-3640b6038868">
<img width="368" alt="image" src="https://github.com/user-attachments/assets/86674a27-6c99-4ceb-8329-b8f2b36c4b76">



We could easily fix it by adding this to `application.tailwind.css`

```
div.field_with_errors {
  @apply w-full;
}
```